server/db/migrations/20220622125257-modify_items_add_complete.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		return queryInterface.addColumn("Items", "status", {
```

Comment:
note that your previous migration already contains a status column so this migration fails when running it